rpc/connection.go

```diff
@@ -609,6 +609,9 @@ func (c *Connection) doReconnect(ctx context.Context, disconnectStatus Disconnec
 			// A fatal error happened.
 			*reconnectErrPtr = err
 			// short-circuit Retry
+			if err != nil {
+				c.transport.Close()
```

Comment:
I don't know about that error, but Gregor returns other errors from there that could pile up. I never actually have seen 255 happen, but I speculated that this issue could be a cause. I've only seen this problem with the Gregor connection, not on kbfsfuse.